.ks-serverless/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+source .env
+echo 'START BUILD'
+npm i
+#dependency needed to build kapi avoid uploading node_modules
+npm i uws@10.148.1 # https://github.com/socketio/socket.io/issues/3249
+echo 'SUCCESSS ADD DEPENDENCIES'
+npm run kapi:serverless
+webpack --config .ks-serverless/webpack.config.js # need installed webpack and webpack-cli
```

Comment:
what if we add the `--optimize-minimize` to apply tree shaking?
makes sense?

```suggestion
webpack --optimize-minimize --config .ks-serverless/webpack.config.js # need installed webpack and webpack-cli
```

Reply:
we are using the ` mode: 'production',` configuration on webapack, so it does *not* makes sense xD

Reply:
For sure we can do that. I'm not expert on webpack so go ahead and make it better.

Reply:
It is ok as it is right now :)